internal/changestream/testing/modelsuite.go

```diff
@@ -61,3 +70,27 @@ func (s *ModelSuite) AssertChangeStreamIdle(c *gc.C) {
 		}
 	}
 }
+
+// PrimeChangeStream the change stream with some initial data. This ensures
+// that the change stream has some initial data otherwise the upper bound
+// won't be set correctly. The model database has no triggers for the initial
```

Comment:
Could such a trigger be used by a gate style watcher to designate activation?

Reply:
Maybe I'm misunderstanding your question. Since `PrimeChangeStream` will execute on `SetUpTests()` then it shouldn't interfere with watchers that are created on each test.